gunagala/tests/test_psf.py

```diff
@@ -101,8 +108,8 @@ def test_shape(psf_moffat):
 
 
 @pytest.mark.parametrize("psf, image_size", [
-    (moffat, (21, 21)),
-    (pixellated, (21, 21))],
+    (psf_moffat(), (21, 21)),
```

Comment:
I think we can parameterize the parameters. Looks like a lot of repetition here. Are the tests themselves different or just the shapes you pass to them?

Reply:
The thing is, you have an `image_size` parameter but you're just giving it the same value each time, so you're not really using it.

If you're going to have an `image_size` parameter you should be using it to pass different image sizes, which would enable you to combine the `test_pixellated_square`, `test_pixellated_rectangles`, etc. tests into one.